superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -126,16 +126,58 @@ def add_parser_arguments(self):
                     help='Number of threads in %s test.' % io_str,
                 )
 
-    def _preprocess(self):
+        self._parser.add_argument(
+            '--numa_node',
+            type=int,
+            required=False,
+            help='The index of NUMA node.',
+        )
+        self._parser.add_argument(
+            '--verify',
+            type=str,
+            required=False,
+            help='Verification method specified for fio --verify flag.',
+        )
+
+    def __get_arguments_from_env(self):
```

Comment:
I revised this into _get_arguments_from_env, and moved this and args.numa into MicroBenchmarkWithInvoke in another pr, so here you may can need to change the function name, replace the numa_node into numa

Reply:
fixed